dhis-2/dhis-api/src/test/java/org/hisp/dhis/eventhook/targets/auth/ApiTokenAuthSchemeTest.java

```diff
@@ -47,7 +47,7 @@ void testAuthorizationHeaderSet() {
         new ApiTokenAuthScheme().setToken("90619873-3287-4296-8C22-9E1D49C0201F");
 
     MultiValueMap<String, String> headers = new LinkedMultiValueMap<>();
-    auth.apply(headers, null);
+    auth.apply(null, headers, null);
```

Comment:
IDK what is going on here but a generic function being called with 2 `null` values deserves a comment to explain why that is intentional.

Reply:
`null` is fine if it works, I just would leave a comment explaining why null is the value to use and what it means.